Dockerfile

```
@@ -0,0 +1,16 @@
+FROM node:alpine
+
+# Set working directory inside the container
+WORKDIR /usr/src
+
+# Copy package.json and yarn.lock to install dependencies
+COPY package.json yarn.lock ./
+
+# Install dependencies using yarn
+RUN yarn install --ignore-engines || yarn install --ignore-engines
```

Why are you doing that twice?

```suggestion
RUN yarn install --ignore-engines
```